app/assets/stylesheets/home.scss

```diff
@@ -0,0 +1,104 @@
+// Place all the styles related to the home controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+body {
+  background-color: #eff3f6;
+  font-family: Helvetica;
+  color: #3f729b;
+  text-align: center;
+}
+
+a {
+  text-decoration: none;
+  color: #00405d;
+}
+
+.welcome {
+  font-size: 8em;
+  text-align: center;
+  margin-top: 150px;
+}
+
+.registration {
+  width: 600px;
+  background-color: red;
+}
+
+.form {
+  width: 500px;
+  margin: 100px auto;
+  text-align: left;
+
+}
+
+
+.header {
+  height: 60px;
```

Comment:
Properties should be ordered background-color, height, margin-left, margin-right, margin-top, padding, text-align